lib/Sema/TypeChecker.h

```diff
@@ -2068,13 +2068,14 @@ class TypeChecker final : public LazyResolver {
       const Optional<std::function<void()>> &invalidTypeCtxDiagnostic = None);
 
   /// SWIFT_ENABLE_TENSORFLOW
-  /// Creates a `AutoDiffParameterIndices *` for the given function type,
-  /// representing all inferred differentiation parameters.
+  /// Creates a `AutoDiffIndexSubset` for the given function type, representing
```

Comment:
```suggestion
  /// Creates an `AutoDiffIndexSubset` for the given function type, representing
```

Reply:
Done in https://github.com/apple/swift/pull/27514/commits/37d1a911cd33f0a3bc45ce1fe0499727ed04c60a.